uavsar_pytools/rename.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import os
+import glob
+from os import path
+import shutil
+
+def rename(dir_in, clean_dir = False):
+    """This function renames UAVSAR data in a directory containing csv file of annotation to a more intuitive name.
+        The format will be sitename_date-of-first-acquisition_date-of-second-acquisition_XXXpolarization_filetype(coh/amp/hgd).
+        The renamed files are stored in a new directory called renamed.
+
+    Args:
+        dir_in (_type_): path to the directory containing the tiff files to be renamed
+        clean_dir (bool, optional): wether to delete the old files or not. Defaults to False.
+    """    
+
+    #check if crop directory exists
+    dir_out = dir_in + '/renamed/'
+    if not os.path.exists(dir_out):
+        os.makedirs(dir_out)
+
+    #read the metadata from the csv file
+    metadata = pd.read_csv(glob.glob(dir_in + '/*grd.csv')[0]) #it gives a list, so select the only item in the list
```

Comment:
could you from glob import glob and then just use glob here to match the pattern in the rest of the repo?